google/cloud/security/scanner/scanner.py

```diff
@@ -240,15 +224,43 @@ def _write_violations_output(logger, violations):
                 'member': '{}:{}'.format(member.type, member.name)
             }
 
-def _upload_csv_to_gcs(logger, output_path, output_filename, csv_name):
+def _output_results(logger, all_violations, **kwargs):
+    """Send the output results.
+
+    Args:
+        logger: The logger.
```

Comment:
I chose not to make it global. Currently working on extending the log utility to look for commandline flags and things get really ugly (because of gflags) when flags are not parsed before accessing them -- that's what would happen if I had the logger.setup_logging() called on the global scope.